### DISCUSSION TITLE
How to deal with sectors with id 0, OnChain: yes, and active: no?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9827] [Storage Provider] [answered]
At that time, the disk where the chain was located had a slow write fault, which caused more than 70 abnormal sectors. How to deal with it。
![image](https://user-images.githubusercontent.com/39183234/206855849-71dff162-af1e-4f5b-8a43-863338117731.png)

**Comment by Reiers** [comment, marked as answer]
It's an open issue, and should not break anything.

See issue here : https://github.com/filecoin-project/lotus/issues/7779

Feel free to add logs, more information, anything would be really helpful. 
Since it's hard issue to reproduce. 

Thank you !
